src/main/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionController.java

```diff
@@ -80,7 +81,8 @@ public ResponseEntity<RequestTranscriptionResponse> requestTranscription(
 
     @Override
     @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
-    @Authorisation(contextId = TRANSCRIPTION_ID, securityRoles = {APPROVER, TRANSCRIBER})
+    @Authorisation(contextId = TRANSCRIPTION_ID, securityRoles = {APPROVER, TRANSCRIBER},
+        globalAccessSecurityRoles = {})
```

Comment:
Will you be removing the empty `globalAccessSecurityRoles = {}` now that we have a default value in the annotation?